docs/README.md

```diff
@@ -92,6 +92,10 @@ If you want to add a new component, you must follow these steps:
 * If necessary you can add snapshots for it by modifying `react/examples.spec.jsx` and updating them `yarn build && yarn test -u`
 * Remember to propagate the possible `ref` with `React.forwardRef`. [See forwardRef documentation](https://en.reactjs.org/docs/forwarding-refs.html)
 
+### Rename/Move a component
+
+When renaming or moving a component of Cozy-UI, it's important to check that no Cozy partner's code is using this file.
+
```

Comment:
On peut généraliser je pense :)

`When renaming or moving a Cozy-UI component, it may cause a breaking change. In this case, you should provide a codemod as much as possible to fix it.`